schema.md

```diff
@@ -0,0 +1,555 @@
+
+The e-mail address must not be obfuscated. To improve resistance
+against e-mail collection, use `\x64` to replace `@`, as allowed
+by the YAML specification.
+
+### Key `maintenance/technical-contacts/affiliation`
+
+* Type: string
+* Presence: optional
+* Example: "Acme Inc."
+
+This key contains an explicit affiliation information for the technical
+contact. In case of multiple maintainers, this can be used to create
+a relation between each technical contact and each maintainer entity.
+
+
+
+
+
+
+
+
+
+
+
+
+
+## Section `description`
+
+This section contains general description on the software. Parsers
+can use this section for instance to create a web page describing
+the software.
+
+### Key `description/name`
+
+* Type: string
+* Presence: mandatory
+* Example: "Medusa"
+
+This key contains the (short) public name of the product. It should be
+the name most people usually refer to the software. In case the software
+has both an internal "code" name and a commercial name, use the
+commercial name.
+
+### Key `description/version`
+
+* Type: string
+* Presence: optional
+* Example: "1.0", "dev"
+
+This key contains the latest stable version number of the software.
+The version number is a string that is not meant to be interpreted
+and parsed but just displayed; parsers should not assume semantic
+versioning or any other specific version format.
+
+The key can be omitted if the software is currently in initial
+development and has never been released yet.
+
+### Key `description/released`
+
+* Type: string (date)
+* Presence: mandatory if `description/version` is present
+* Example: "2017-04-15"
+
+This key contains the date at which the latest version was released.
+This date is mandatory if the software has been released at least once
+and thus he version number is present.
+
+### Key `description/platforms`
+
+* Type: enumerated string or array of strings
+* Presence: mandatory
+* Values: "web", "windows", "mac", "linux", "ios", "android". Values
+  outside this list are allowed.
+* Example: "web"
+
+This key specifies which platform the software runs on. It is meant
+to describe the platforms that users will use to access and operate
+the software, rather than the platform the software itself runs on.
+
+Use the predefined values if possible. If the software runs on a
+platform for which a predefined value is not available, a different
+value can be used.
+
+### Key `description/logo`
+
+* Type: string or array of strings (path to file)
+* Presence: optional
+* Formats: SVG, PNG
+* Example: "img/logo.svg"
+
+This key contains the logo of the software. Logos should be in vector
+format, but raster formats are allowed as a fallback.
+
+Vector logos should be provided at least in two formats: a color one,
+and a monochromatic (black) one.
+
+Raster logos should be provided only if vector logos do not exist.
+In this case, they should be transparent PNGs, minimum 1000px of
+width.
+
+### Key `description/shortdesc`
```

Comment:
Inconsistent key naming (eg `repo-owner` and `shortdesc`)

Consider renaming this to kebab style `short-description` or renaming other things to `repoowner`.

Reply:
As agreed in the issue, I'll send a separate PR to migrate to camelCase.